### ISSUE TITLE
Admin permissions fail to import

### ISSUE THREAD

**Issue by danielbitzer** [boazpoolman/strapi-plugin-config-sync#42]
Firstly, thank you for building this fantastic plugin!

### Describe the bug

I'm finding that in my production environment admin permissions always fail to import. Other config types are fine. I've tried importing on bootstrap, via CLI and via the GUI. All fail.

Running the import command multiple times doesn't do anything.

<img width="861" alt="Screen Shot 2022-01-21 at 7 43 37 am" src="https://user-images.githubusercontent.com/1892137/150423557-fb094c32-8bfc-488b-a081-64dba7682593.png">

The diff is visible in the GUI.

<img width="832" alt="Screen Shot 2022-01-21 at 7 48 21 am" src="https://user-images.githubusercontent.com/1892137/150423637-f4b53169-f235-45d9-92df-a77bef0cc8b9.png">

### My config
```
'config-sync': {
    enabled: true,
    config: {
      importOnBootstrap: true,
      excludedConfig: [
        "core-store.plugin_users-permissions_advanced"
      ],
    },
  },
```

### System

- Node.js version: 12.22.1
- NPM version: 6.14.12
- Strapi version: 4.0.5
- Plugin version: 1.0.0-beta.5
- Database: Mysql
- Operating system: Ubuntu 20.4

**Comment by boazpoolman** [comment]
Hey @danielbitzer,

Thank you for the issue submission!

I think this is an issue with the ordering of the file contents.

Can you confirm in the diff viewer in the GUI that the changes are actually only the order of the data that is different?

**Comment by boazpoolman** [comment]
So I THINK that if this is an issue of order, a have a fix for it.
You could try it out in your project like so:

```
yarn add boazpoolman/strapi-plugin-config-sync#pull/43/head
npm install boazpoolman/strapi-plugin-config-sync#pull/43/head
```

**To test:**:

- Install PR locally
- Export config locally
- Push config to git
- Pull config to production
- See if you still have issues with importing

**Comment by danielbitzer** [comment]
Thanks for looking into this so quickly! 

I have tried #43 and initially it seemed to work but then I made a couple of other changes and they don't seem to be importing when they get to production.

Many of the changes in the diff _are_ simply re-ordered. I'm not intentionally re-ordering the fields. This time I've added permissions for one field (`abn`) and the diff is as follows:

<img width="869" alt="Screen Shot 2022-01-24 at 4 42 13 pm" src="https://user-images.githubusercontent.com/1892137/150731265-8e97109d-bf6e-403f-ab23-d76449bfd2fa.png">

I'm not quite sure why all the fields keep re-ordering but the changes are not just for re-ordering.

**Comment by boazpoolman** [comment]
Allright. So one bug is fixed; the incorrectly ordering of the relations.
Now the next one is; the incorrectly ordering of the the field content of a specific relation (`properties` in this case)

I need to look in to this more closely. Please be patient with me.

Conclusion based on your screenshot:
The `properties` are alphabetically ordered in the files but not in the database.

**Comment by boazpoolman** [comment]
Ah I think I see the issue.
As of now; relations can only be created or deleted when you import.
It can't be updated, which is what you are trying to do here.

**Comment by boazpoolman** [comment]
@danielbitzer Just to be sure my assumption is correct:

The changes you have open now are only changes to the `properties` or `conditions`?
No changes to any `action` or `subject` like there were in the first screenshot of the issue.

**Comment by danielbitzer** [comment]
Amazing! That's working great now. Thank you! Do you have a donate link?

Side note: I did encounter a separate issue but I'm not quite sure where the fault was. It may have been Strapi or user error. I made several changes to content type fields and the view config and I was getting the white screen of death for certain config types. The issue disappeared when I disabled the config-sync plugin. But I was able to resolve the issue by deleting my config sync directory and then doing a fresh export.

**Comment by boazpoolman** [comment]
@danielbitzer Just had contact with another person having this issue.
For him the problem was he had `importOnBootstrap` enabled locally, which is not recommended.

I'm not sure if that's the issue on your side as wel. But if it is you can do this to resolve it:

`config/plugins.js`
```
module.exports = ({ env }) => ({
    'config-sync': {
      enabled: true,
      config: {
        importOnBootstrap: env('NODE_ENV') !== 'development',
      }
    },
});
```

**Comment by danielbitzer** [comment]
> Released with v1.0.0-beta.6.

Great! Thank you!

> I've seen somebody who had a similar issue and it turned out to be strapi/strapi#11996.

Interesting, thanks for the pointer.

> For him the problem was he had `importOnBootstrap` enabled locally, which is not recommended.

I literally discovered I'd made this mistake yesterday. I think that's a good change to prevent it happening in the dev environment.

**Comment by boazpoolman** [comment]
Thank you for the donation @danielbitzer!
Very much appreciated ❤️
